libs/spa/view/dashboard/src/lib/dashboard.component.css

```diff
@@ -17,6 +17,10 @@ nz-layout {
 			color: var(--ant-primary-color);
 			font-size: 24px;
 		}
+
+		nz-avatar {
+			cursor: pointer;
```

Comment:
Why should the avatar have a pointer cursor? Isn't it only clickable if it's part of a button?